### PR TITLE
feat: Make color configurable

### LINKED ISSUE #4: Make color optionally configurable
It would be nice to be able to specify the color of a song.

### Definition of done
The song objects in `window.songs` accept an optional `colour` field which when set will be the color coordinates of the rendered song cell.

### PR DESCRIPTION
**Pull request by netrome**
Closes #4

### CODE REVIEW

File: scripts/string_color.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+  str.split("").forEach((char) => {
+    hash = char.charCodeAt(0) + ((hash << 5) - hash);
+  });
+  let colour = "#";
+  for (let i = 0; i < 2; i++) {
+    const value = (hash >> (i * 8)) & 0xff;
+    colour += "0" + value.toString(16).padStart(1, "0");
+  }
+  return colour + "ff";
+}
```
Comment (netrome) [inline scripts/string_color.js:16]: From `deno fmt`

File: scripts/string_color.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
+function songToColour(song) {
+  return song.colour ?? stringToColour(song.feeling);
```
Comment (netrome) [inline scripts/string_color.js:2]: TIL about the ?? ES2020 Nullish coalescing operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Comment (dtennander) [reply scripts/string_color.js:2]: Let’s hope no one in FriRoll still uses IE 🙈

**Review by dtennander — APPROVED** [review]
LGTM! 🎉

**Pull request merged**